### DISCUSSION TITLE
通过域名和端口号无法通过外网访问alist,请问是什么原因？

### DISCUSSION THREAD

**Discussion by lbenw126** [AlistGo/alist#4990] [Q&A]
通过docker挂载的alist,通过内网ip加端口号能正常访问，最近突然发现外网通过域名加端口号无法访问了，域名也能ping通返回正确ip地址，通过外网ip加端口号也能访问，请问是什么原因？

**Comment by mingtianshi** [comment]
啊这，有没有可能你需要的是NGINX
